compiler/src/test/scala/org/scalajs/nscplugin/test/util/VersionDependentUtils.scala

```diff
@@ -16,19 +16,14 @@ object VersionDependentUtils {
   val scalaVersion = scala.util.Properties.versionNumberString
 
   /** Does the current Scala version support the `@nowarn` annotation? */
-  val scalaSupportsNoWarn = {
-    !scalaVersion.startsWith("2.12.") &&
-    scalaVersion != "2.13.0" &&
-    scalaVersion != "2.13.1"
-  }
+  val scalaSupportsNoWarn =
+    !scalaVersion.startsWith("2.12.")
 
   private val usesColonInMethodSig = {
     /* Yes, this is the same test as in scalaSupportsNoWarn, but that's
      * completely coincidental, so we have a copy.
      */
-    !scalaVersion.startsWith("2.12.") &&
-    scalaVersion != "2.13.0" &&
-    scalaVersion != "2.13.1"
+    !scalaVersion.startsWith("2.12.")
   }
```

Comment:
Maybe at this point it makes more sense to:

```scala
private val isScala212 = scalaVersion.startsWith("2.12.")

val scalaSupportsNoWarn = !isScala212
```

(and maybe even remove `usesColonInMethodSig` and just use `isScala212`).